person.rb

```diff
@@ -1,7 +1,7 @@
 require_relative('./nameable')
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] rentals should also have  a setter and a getter, kindly add it to `attr_accessor` 
e.g
```
`attr_accessor :name, :age, :rentals`
```